src/locales/fr-fr.ts

```diff
@@ -102,13 +102,13 @@ const frFR: Partial<Locale> = {
     skipLinkLabel: () => "Passer au contenu principal",
   },
   loader: {
-    loading: () => "Téléhargement",
+    loading: () => "Chargement en cours",
```

Comment:
**question**: Thanks for raising this @arvig33, I was just wondering has this change been approved/provided by a member of the translations team?